src/LcdMenu.cpp

```
@@ -13,14 +20,28 @@ void LcdMenu::setScreen(MenuScreen* screen) {
+
+    FrameLifecycleRenderer* frame = frameLifecycle(renderer);
+    if (frame != NULL) {
+        frame->endFrame();
+    }
 }
 
 bool LcdMenu::process(const unsigned char c) {
     if (!enabled) {
         return false;
     }
     renderer.restartTimer();
```

Cppcheck (2.20.0)

[style] The function 'remove' is never used.

(unusedFunction)